src/main/java/io/javalin/plugin/openapi/handler/DocumentedCrudHandler.kt

```diff
@@ -1,19 +1,23 @@
-package io.javalin.plugin.openapi.dsl
+package io.javalin.plugin.openapi.handler
 
 import io.javalin.apibuilder.CrudHandler
+import io.javalin.apibuilder.CrudHandlerType
 import io.javalin.http.Context
+import io.javalin.http.Handler
+import io.javalin.plugin.openapi.dsl.OpenApiCrudHandlerDocumentation
 
-class DocumentedCrudHandler(
+data class DocumentedCrudHandler(
         val crudHandlerDocumentation: OpenApiCrudHandlerDocumentation,
         private val crudHandler: CrudHandler
 ) : CrudHandler {
```

Comment:
Becomes a "normal" crud handler now, since the annotations are inferred by the refrences.